From d7d06b6c391deb511533392d56f799b155f99677 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Thu, 24 Aug 2017 05:11:16 -0700
Subject: [PATCH 2616/2639] Branching for 5.13.1-SNAPSHOT on Thu Aug 24
 05:10:35 PDT 2017

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '492'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5'

Pushed to remote common	git@github.sf.cloudera.com:Mirrors/apache-hadoop-common.git (push)
origin	git@github.sf.cloudera.com:CDH/hadoop.git (push)
---
 hadoop-assemblies/pom.xml                          |    4 ++--
 hadoop-build-tools/pom.xml                         |    2 +-
 hadoop-client/pom.xml                              |    4 ++--
 hadoop-common-project/hadoop-annotations/pom.xml   |    4 ++--
 hadoop-common-project/hadoop-auth-examples/pom.xml |    4 ++--
 hadoop-common-project/hadoop-auth/pom.xml          |    4 ++--
 hadoop-common-project/hadoop-common/pom.xml        |    4 ++--
 hadoop-common-project/hadoop-kms/pom.xml           |    4 ++--
 hadoop-common-project/hadoop-minikdc/pom.xml       |    4 ++--
 hadoop-common-project/hadoop-nfs/pom.xml           |    4 ++--
 hadoop-common-project/pom.xml                      |    4 ++--
 hadoop-dist/pom.xml                                |    4 ++--
 hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml     |    4 ++--
 hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml        |    4 ++--
 hadoop-hdfs-project/hadoop-hdfs/pom.xml            |    4 ++--
 .../hadoop-hdfs/src/contrib/bkjournal/pom.xml      |    4 ++--
 hadoop-hdfs-project/pom.xml                        |    4 ++--
 .../hadoop-mapreduce-client-app/pom.xml            |    4 ++--
 .../hadoop-mapreduce-client-common/pom.xml         |    4 ++--
 .../hadoop-mapreduce-client-core/pom.xml           |    4 ++--
 .../hadoop-mapreduce-client-hs-plugins/pom.xml     |    4 ++--
 .../hadoop-mapreduce-client-hs/pom.xml             |    4 ++--
 .../hadoop-mapreduce-client-jobclient/pom.xml      |    4 ++--
 .../hadoop-mapreduce-client-nativetask/pom.xml     |    4 ++--
 .../hadoop-mapreduce-client-shuffle/pom.xml        |    4 ++--
 .../hadoop-mapreduce-client/pom.xml                |    4 ++--
 .../hadoop-mapreduce-examples/pom.xml              |    4 ++--
 hadoop-mapreduce-project/pom.xml                   |    4 ++--
 hadoop-mapreduce1-project/build.xml                |    2 +-
 hadoop-mapreduce1-project/cloudera-pom.xml         |    4 ++--
 .../cloudera/maven-packaging/hadoop-client/pom.xml |    4 ++--
 .../cloudera/maven-packaging/hadoop-core/pom.xml   |    4 ++--
 .../maven-packaging/hadoop-examples/pom.xml        |    4 ++--
 .../maven-packaging/hadoop-minicluster/pom.xml     |    4 ++--
 .../cloudera/maven-packaging/hadoop-mrunit/pom.xml |    4 ++--
 .../maven-packaging/hadoop-streaming/pom.xml       |    4 ++--
 .../maven-packaging/hadoop-tarball/pom.xml         |    4 ++--
 .../cloudera/maven-packaging/hadoop-test/pom.xml   |    4 ++--
 .../cloudera/maven-packaging/hadoop-tools/pom.xml  |    4 ++--
 .../cloudera/maven-packaging/pom.xml               |    4 ++--
 hadoop-mapreduce1-project/ivy/libraries.properties |    4 ++--
 hadoop-maven-plugins/pom.xml                       |    2 +-
 hadoop-minicluster/pom.xml                         |    4 ++--
 hadoop-project-dist/pom.xml                        |    4 ++--
 hadoop-project/pom.xml                             |    4 ++--
 hadoop-tools/hadoop-ant/pom.xml                    |    4 ++--
 hadoop-tools/hadoop-archive-logs/pom.xml           |    4 ++--
 hadoop-tools/hadoop-archives/pom.xml               |    4 ++--
 hadoop-tools/hadoop-aws/pom.xml                    |    4 ++--
 hadoop-tools/hadoop-azure-datalake/pom.xml         |    2 +-
 hadoop-tools/hadoop-azure/pom.xml                  |    4 ++--
 hadoop-tools/hadoop-datajoin/pom.xml               |    4 ++--
 hadoop-tools/hadoop-distcp/pom.xml                 |    4 ++--
 hadoop-tools/hadoop-extras/pom.xml                 |    4 ++--
 hadoop-tools/hadoop-gridmix/pom.xml                |    4 ++--
 hadoop-tools/hadoop-openstack/pom.xml              |    4 ++--
 hadoop-tools/hadoop-pipes/pom.xml                  |    4 ++--
 hadoop-tools/hadoop-rumen/pom.xml                  |    4 ++--
 hadoop-tools/hadoop-sls/pom.xml                    |    4 ++--
 hadoop-tools/hadoop-streaming/pom.xml              |    4 ++--
 hadoop-tools/hadoop-tools-dist/pom.xml             |    4 ++--
 hadoop-tools/pom.xml                               |    4 ++--
 .../hadoop-yarn/hadoop-yarn-api/pom.xml            |    4 ++--
 .../pom.xml                                        |    4 ++--
 .../pom.xml                                        |    4 ++--
 .../hadoop-yarn/hadoop-yarn-applications/pom.xml   |    4 ++--
 .../hadoop-yarn/hadoop-yarn-client/pom.xml         |    4 ++--
 .../hadoop-yarn/hadoop-yarn-common/pom.xml         |    4 ++--
 .../hadoop-yarn/hadoop-yarn-registry/pom.xml       |    4 ++--
 .../pom.xml                                        |    4 ++--
 .../hadoop-yarn-server-common/pom.xml              |    4 ++--
 .../hadoop-yarn-server-nodemanager/pom.xml         |    4 ++--
 .../hadoop-yarn-server-resourcemanager/pom.xml     |    4 ++--
 .../hadoop-yarn-server-tests/pom.xml               |    4 ++--
 .../hadoop-yarn-server-web-proxy/pom.xml           |    4 ++--
 .../hadoop-yarn/hadoop-yarn-server/pom.xml         |    4 ++--
 .../hadoop-yarn/hadoop-yarn-site/pom.xml           |    4 ++--
 hadoop-yarn-project/hadoop-yarn/pom.xml            |    4 ++--
 hadoop-yarn-project/pom.xml                        |    4 ++--
 pom.xml                                            |    4 ++--
 80 files changed, 156 insertions(+), 156 deletions(-)

diff --git a/hadoop-assemblies/pom.xml b/hadoop-assemblies/pom.xml
index d9915bb..928c3c6 100644
--- a/hadoop-assemblies/pom.xml
+++ b/hadoop-assemblies/pom.xml
@@ -23,12 +23,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-assemblies</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>Apache Hadoop Assemblies</name>
   <description>Apache Hadoop Assemblies</description>
 
diff --git a/hadoop-build-tools/pom.xml b/hadoop-build-tools/pom.xml
index 1a5911a..6c7e314 100644
--- a/hadoop-build-tools/pom.xml
+++ b/hadoop-build-tools/pom.xml
@@ -18,7 +18,7 @@
   <parent>
     <artifactId>hadoop-main</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>hadoop-build-tools</artifactId>
diff --git a/hadoop-client/pom.xml b/hadoop-client/pom.xml
index 6ffbf0e..2b0f265 100644
--- a/hadoop-client/pom.xml
+++ b/hadoop-client/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-client</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Client</description>
diff --git a/hadoop-common-project/hadoop-annotations/pom.xml b/hadoop-common-project/hadoop-annotations/pom.xml
index f0d95c0..958e2dc 100644
--- a/hadoop-common-project/hadoop-annotations/pom.xml
+++ b/hadoop-common-project/hadoop-annotations/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-annotations</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Annotations</description>
   <name>Apache Hadoop Annotations</name>
   <packaging>jar</packaging>
diff --git a/hadoop-common-project/hadoop-auth-examples/pom.xml b/hadoop-common-project/hadoop-auth-examples/pom.xml
index adbd255..33384bb 100644
--- a/hadoop-common-project/hadoop-auth-examples/pom.xml
+++ b/hadoop-common-project/hadoop-auth-examples/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-auth-examples</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>war</packaging>
 
   <name>Apache Hadoop Auth Examples</name>
diff --git a/hadoop-common-project/hadoop-auth/pom.xml b/hadoop-common-project/hadoop-auth/pom.xml
index 77dd428..5b0ab5c 100644
--- a/hadoop-common-project/hadoop-auth/pom.xml
+++ b/hadoop-common-project/hadoop-auth/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-auth</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <name>Apache Hadoop Auth</name>
diff --git a/hadoop-common-project/hadoop-common/pom.xml b/hadoop-common-project/hadoop-common/pom.xml
index c8fcbd1..46a982f 100644
--- a/hadoop-common-project/hadoop-common/pom.xml
+++ b/hadoop-common-project/hadoop-common/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-common</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Common</description>
   <name>Apache Hadoop Common</name>
   <packaging>jar</packaging>
diff --git a/hadoop-common-project/hadoop-kms/pom.xml b/hadoop-common-project/hadoop-kms/pom.xml
index c3b94ce..1900c3c 100644
--- a/hadoop-common-project/hadoop-kms/pom.xml
+++ b/hadoop-common-project/hadoop-kms/pom.xml
@@ -22,12 +22,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-kms</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>war</packaging>
 
   <name>Apache Hadoop KMS</name>
diff --git a/hadoop-common-project/hadoop-minikdc/pom.xml b/hadoop-common-project/hadoop-minikdc/pom.xml
index 35bddd7..7aeb30e 100644
--- a/hadoop-common-project/hadoop-minikdc/pom.xml
+++ b/hadoop-common-project/hadoop-minikdc/pom.xml
@@ -18,13 +18,13 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minikdc</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop MiniKDC</description>
   <name>Apache Hadoop MiniKDC</name>
   <packaging>jar</packaging>
diff --git a/hadoop-common-project/hadoop-nfs/pom.xml b/hadoop-common-project/hadoop-nfs/pom.xml
index f757d07..9ce4e7a 100644
--- a/hadoop-common-project/hadoop-nfs/pom.xml
+++ b/hadoop-common-project/hadoop-nfs/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-nfs</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <name>Apache Hadoop NFS</name>
diff --git a/hadoop-common-project/pom.xml b/hadoop-common-project/pom.xml
index b0a527a..699c6bc 100644
--- a/hadoop-common-project/pom.xml
+++ b/hadoop-common-project/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-common-project</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Common Project</description>
   <name>Apache Hadoop Common Project</name>
   <packaging>pom</packaging>
diff --git a/hadoop-dist/pom.xml b/hadoop-dist/pom.xml
index c3bd28e..8b7f0cc 100644
--- a/hadoop-dist/pom.xml
+++ b/hadoop-dist/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-dist</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Distribution</description>
   <name>Apache Hadoop Distribution</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
index f125fb6..eb44c81 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
@@ -22,12 +22,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs-httpfs</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>war</packaging>
 
   <name>Apache Hadoop HttpFS</name>
diff --git a/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml
index 67cd705..66ea504 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml
@@ -20,12 +20,12 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs-nfs</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop HDFS-NFS</description>
   <name>Apache Hadoop HDFS-NFS</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/hadoop-hdfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs/pom.xml
index 8864492..89410a0 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/pom.xml
@@ -20,12 +20,12 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop HDFS</description>
   <name>Apache Hadoop HDFS</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml b/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml
index 055f1f2..1d354b1 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml
@@ -20,13 +20,13 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../../../../hadoop-project</relativePath>
   </parent>
 
   <groupId>org.apache.hadoop.contrib</groupId>
   <artifactId>hadoop-hdfs-bkjournal</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop HDFS BookKeeper Journal</description>
   <name>Apache Hadoop HDFS BookKeeper Journal</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/pom.xml b/hadoop-hdfs-project/pom.xml
index 66ddc0a..137d281 100644
--- a/hadoop-hdfs-project/pom.xml
+++ b/hadoop-hdfs-project/pom.xml
@@ -20,12 +20,12 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs-project</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop HDFS Project</description>
   <name>Apache Hadoop HDFS Project</name>
   <packaging>pom</packaging>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml
index 954ed8d..7fe5b7c 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-app</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-app</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml
index 16f5904..902d056 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-common</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-common</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml
index 5db7b5a..64cf651 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-core</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-core</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml
index 9b804f9..8208e08 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-hs-plugins</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-hs-plugins</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml
index 5d1037e..2ad30dc 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-hs</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-hs</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml
index 33be2f2..5e527a0 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-jobclient</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml
index 8dce1c5..56efcb2 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-nativetask</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-nativetask</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml
index abbfd50..4c71805 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-shuffle</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client-shuffle</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml
index 87f2ec4..d5621c8 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-mapreduce-client</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml
index 8c3b2d2..a8179b0 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-examples</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop MapReduce Examples</description>
   <name>Apache Hadoop MapReduce Examples</name>
   <packaging>jar</packaging>
diff --git a/hadoop-mapreduce-project/pom.xml b/hadoop-mapreduce-project/pom.xml
index 25e5577..9709252 100644
--- a/hadoop-mapreduce-project/pom.xml
+++ b/hadoop-mapreduce-project/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>hadoop-mapreduce</name>
   <url>http://hadoop.apache.org/mapreduce/</url>
diff --git a/hadoop-mapreduce1-project/build.xml b/hadoop-mapreduce1-project/build.xml
index 9e002c2..bb4f797 100644
--- a/hadoop-mapreduce1-project/build.xml
+++ b/hadoop-mapreduce1-project/build.xml
@@ -54,7 +54,7 @@
  
   <property name="Name" value="Hadoop"/>
   <property name="name" value="hadoop"/>
-  <property name="version" value="2.6.0-mr1-cdh5.13.0-SNAPSHOT"/>
+  <property name="version" value="2.6.0-mr1-cdh5.13.1-SNAPSHOT"/>
   <property name="final.name" value="${name}-${version}"/>
   <property name="test.final.name" value="${name}-test-${version}"/>
   <property name="year" value="2009"/>
diff --git a/hadoop-mapreduce1-project/cloudera-pom.xml b/hadoop-mapreduce1-project/cloudera-pom.xml
index 451dfd1..7084003 100644
--- a/hadoop-mapreduce1-project/cloudera-pom.xml
+++ b/hadoop-mapreduce1-project/cloudera-pom.xml
@@ -26,13 +26,13 @@ NOTE - This POM is used for Cloudera's build and packaging process.
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>cloudera/maven-packaging/pom.xml</relativePath>
   </parent>
   
   <groupId>com.cloudera.cdh</groupId>
   <artifactId>hadoop-ant</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <description>Maven wrapper for Ant Hadoop build</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
index 2a2337e..9482f3d 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-client</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Client</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
index d6a4e13..0fee0f2 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
@@ -23,13 +23,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-core</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-core</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
index d71ecfd..fde4a0d 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-examples</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-examples</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
index e423345..c2a255e 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minicluster</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Mini-Cluster</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
index 44cb871..820c548 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mrunit</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-mrunit</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
index d774b62..bd48cc2 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-streaming</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-streaming</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
index 89f5127..670f10c 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tarball</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   
   <name>hadoop-tarball</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
index 4c63d80..8a873f4 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-test</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   
   <name>hadoop-test</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
index 9e4c263..643f356 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   
   <name>hadoop-tools</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
index 0139650..6f31f56 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
@@ -21,13 +21,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.13.0-SNAPSHOT</version>
+    <version>5.13.1-SNAPSHOT</version>
     <relativePath>../../../../../pom.xml</relativePath>
   </parent>
   
   <groupId>com.cloudera.cdh</groupId>
   <artifactId>hadoop-root</artifactId>
-  <version>2.6.0-mr1-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <description>Hadoop root</description>
diff --git a/hadoop-mapreduce1-project/ivy/libraries.properties b/hadoop-mapreduce1-project/ivy/libraries.properties
index f1e6163..c910ef4 100644
--- a/hadoop-mapreduce1-project/ivy/libraries.properties
+++ b/hadoop-mapreduce1-project/ivy/libraries.properties
@@ -42,8 +42,8 @@ ftplet-api.version=1.0.0
 ftpserver-core.version=1.0.0
 ftpserver-deprecated.version=1.0.0-M2
 
-hadoop-common.version=2.6.0-cdh5.13.0-SNAPSHOT
-hadoop-hdfs.version=2.6.0-cdh5.13.0-SNAPSHOT
+hadoop-common.version=2.6.0-cdh5.13.1-SNAPSHOT
+hadoop-hdfs.version=2.6.0-cdh5.13.1-SNAPSHOT
 
 hsqldb.version=1.8.0.10
 
diff --git a/hadoop-maven-plugins/pom.xml b/hadoop-maven-plugins/pom.xml
index 6e6517d..c0719e4 100644
--- a/hadoop-maven-plugins/pom.xml
+++ b/hadoop-maven-plugins/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
diff --git a/hadoop-minicluster/pom.xml b/hadoop-minicluster/pom.xml
index cd56eb8..080d4f3 100644
--- a/hadoop-minicluster/pom.xml
+++ b/hadoop-minicluster/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minicluster</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Mini-Cluster</description>
diff --git a/hadoop-project-dist/pom.xml b/hadoop-project-dist/pom.xml
index c636409..47c0262 100644
--- a/hadoop-project-dist/pom.xml
+++ b/hadoop-project-dist/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-project-dist</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Project Dist POM</description>
   <name>Apache Hadoop Project Dist POM</name>
   <packaging>pom</packaging>
diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 750099d..3f0a31d 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -20,11 +20,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-main</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-project</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Project POM</description>
   <name>Apache Hadoop Project POM</name>
   <packaging>pom</packaging>
diff --git a/hadoop-tools/hadoop-ant/pom.xml b/hadoop-tools/hadoop-ant/pom.xml
index 84dbbdb..bb429dc 100644
--- a/hadoop-tools/hadoop-ant/pom.xml
+++ b/hadoop-tools/hadoop-ant/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-ant</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Ant Tasks</description>
   <name>Apache Hadoop Ant Tasks</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-archive-logs/pom.xml b/hadoop-tools/hadoop-archive-logs/pom.xml
index 6e4bca2..bf223c9 100644
--- a/hadoop-tools/hadoop-archive-logs/pom.xml
+++ b/hadoop-tools/hadoop-archive-logs/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-archive-logs</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Archive Logs</description>
   <name>Apache Hadoop Archive Logs</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-archives/pom.xml b/hadoop-tools/hadoop-archives/pom.xml
index 983656b..cd657eb 100644
--- a/hadoop-tools/hadoop-archives/pom.xml
+++ b/hadoop-tools/hadoop-archives/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-archives</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Archives</description>
   <name>Apache Hadoop Archives</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index 6749f40..70b7cc0 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <artifactId>hadoop-aws</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>Apache Hadoop Amazon Web Services support</name>
   <description>
     This module contains code to support integration with Amazon Web Services.
diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index 53f5469..6485140 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
diff --git a/hadoop-tools/hadoop-azure/pom.xml b/hadoop-tools/hadoop-azure/pom.xml
index f732667..06b8a50 100644
--- a/hadoop-tools/hadoop-azure/pom.xml
+++ b/hadoop-tools/hadoop-azure/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <artifactId>hadoop-azure</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>Apache Hadoop Azure support</name>
   <description>
     This module contains code to support integration with Azure.
diff --git a/hadoop-tools/hadoop-datajoin/pom.xml b/hadoop-tools/hadoop-datajoin/pom.xml
index 622682b..addc145 100644
--- a/hadoop-tools/hadoop-datajoin/pom.xml
+++ b/hadoop-tools/hadoop-datajoin/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-datajoin</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Data Join</description>
   <name>Apache Hadoop Data Join</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-distcp/pom.xml b/hadoop-tools/hadoop-distcp/pom.xml
index ce0d02c..ddea24a 100644
--- a/hadoop-tools/hadoop-distcp/pom.xml
+++ b/hadoop-tools/hadoop-distcp/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-distcp</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Distributed Copy</description>
   <name>Apache Hadoop Distributed Copy</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-extras/pom.xml b/hadoop-tools/hadoop-extras/pom.xml
index b35bf1d..dab8dcd 100644
--- a/hadoop-tools/hadoop-extras/pom.xml
+++ b/hadoop-tools/hadoop-extras/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-extras</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Extras</description>
   <name>Apache Hadoop Extras</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-gridmix/pom.xml b/hadoop-tools/hadoop-gridmix/pom.xml
index 3176382..5ce5004 100644
--- a/hadoop-tools/hadoop-gridmix/pom.xml
+++ b/hadoop-tools/hadoop-gridmix/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-gridmix</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Gridmix</description>
   <name>Apache Hadoop Gridmix</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-openstack/pom.xml b/hadoop-tools/hadoop-openstack/pom.xml
index 7a04763..486480c 100644
--- a/hadoop-tools/hadoop-openstack/pom.xml
+++ b/hadoop-tools/hadoop-openstack/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <artifactId>hadoop-openstack</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>Apache Hadoop OpenStack support</name>
   <description>
     This module contains code to support integration with OpenStack.
diff --git a/hadoop-tools/hadoop-pipes/pom.xml b/hadoop-tools/hadoop-pipes/pom.xml
index f46053e..350b40a 100644
--- a/hadoop-tools/hadoop-pipes/pom.xml
+++ b/hadoop-tools/hadoop-pipes/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-pipes</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Pipes</description>
   <name>Apache Hadoop Pipes</name>
   <packaging>pom</packaging>
diff --git a/hadoop-tools/hadoop-rumen/pom.xml b/hadoop-tools/hadoop-rumen/pom.xml
index e7afd06..0d433fb 100644
--- a/hadoop-tools/hadoop-rumen/pom.xml
+++ b/hadoop-tools/hadoop-rumen/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-rumen</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Rumen</description>
   <name>Apache Hadoop Rumen</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-sls/pom.xml b/hadoop-tools/hadoop-sls/pom.xml
index e8f38e1..8fbcf50 100644
--- a/hadoop-tools/hadoop-sls/pom.xml
+++ b/hadoop-tools/hadoop-sls/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-sls</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Scheduler Load Simulator</description>
   <name>Apache Hadoop Scheduler Load Simulator</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-streaming/pom.xml b/hadoop-tools/hadoop-streaming/pom.xml
index 45693ad..7ecdd3e 100644
--- a/hadoop-tools/hadoop-streaming/pom.xml
+++ b/hadoop-tools/hadoop-streaming/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-streaming</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop MapReduce Streaming</description>
   <name>Apache Hadoop MapReduce Streaming</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-tools-dist/pom.xml b/hadoop-tools/hadoop-tools-dist/pom.xml
index b4fdbfc..7f0357e 100644
--- a/hadoop-tools/hadoop-tools-dist/pom.xml
+++ b/hadoop-tools/hadoop-tools-dist/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools-dist</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Tools Dist</description>
   <name>Apache Hadoop Tools Dist</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/pom.xml b/hadoop-tools/pom.xml
index c2ddaa4..54a80bb 100644
--- a/hadoop-tools/pom.xml
+++ b/hadoop-tools/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Tools</description>
   <name>Apache Hadoop Tools</name>
   <packaging>pom</packaging>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml
index 4fff63f..5505ccc 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-api</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-api</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml
index 7c755dd..6907eb3 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-applications</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-applications-distributedshell</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-applications-distributedshell</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml
index 4851b6b..b9efff0 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-applications</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-applications-unmanaged-am-launcher</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-applications-unmanaged-am-launcher</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml
index 81da17b..0e2b5d9 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-applications</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-applications</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml
index 0417a80..73f7f37 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml
@@ -17,11 +17,11 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-client</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-client</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml
index 7f730e1..cad5079 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-common</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-common</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml
index 63495be..f2beac9 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>hadoop-yarn-registry</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-registry</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml
index c3e53b3..82d2837 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml
@@ -22,12 +22,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-applicationhistoryservice</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server-applicationhistoryservice</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml
index 42183d9..3dda516 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-common</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server-common</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml
index 35d37b3..c1a1aaf 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-nodemanager</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server-nodemanager</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
index 7e83121..3ecf695 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-resourcemanager</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server-resourcemanager</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml
index dc1a9fb..3ecb2ca 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-tests</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server-tests</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml
index 34f239a..851db44 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-web-proxy</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server-web-proxy</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml
index dc2f450..9288aff 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-server</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml
index a6d719b..8b0406c 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-site</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <name>hadoop-yarn-site</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-yarn-project/hadoop-yarn/pom.xml b/hadoop-yarn-project/hadoop-yarn/pom.xml
index 935a263..c31ed9c 100644
--- a/hadoop-yarn-project/hadoop-yarn/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/pom.xml
@@ -16,12 +16,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>hadoop-yarn</name>
 
diff --git a/hadoop-yarn-project/pom.xml b/hadoop-yarn-project/pom.xml
index b44599d..f24bfaf 100644
--- a/hadoop-yarn-project/pom.xml
+++ b/hadoop-yarn-project/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+    <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-project</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>hadoop-yarn-project</name>
   <url>http://hadoop.apache.org/yarn/</url>
diff --git a/pom.xml b/pom.xml
index 741c660..7d7b5b0 100644
--- a/pom.xml
+++ b/pom.xml
@@ -19,11 +19,11 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.13.0-SNAPSHOT</version>
+    <version>5.13.1-SNAPSHOT</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-main</artifactId>
-  <version>2.6.0-cdh5.13.0-SNAPSHOT</version>
+  <version>2.6.0-cdh5.13.1-SNAPSHOT</version>
   <description>Apache Hadoop Main</description>
   <name>Apache Hadoop Main</name>
   <packaging>pom</packaging>
-- 
1.7.9.5

